packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/app/initialization/app_setup.dart

```diff
@@ -15,6 +15,8 @@ import 'package:flutter/foundation.dart';{{/push_notifications}}
 import '../../utils/helpers.dart';{{#push_notifications}}
 import 'firebase_messaging_callbacks.dart';{{/push_notifications}}
 
+// ignore_for_file: unused_import
```

Comment:
The reason for the ignore in several places is because the imports are required for this file in order for the developer to finish the implementation (such places are marked with TODOs). Due to the lack of those implementations, the analyzer is reporting that there is no usage of a package and thus displaying a warning. 

On the other side, removing them completely would mean that they have to manually add those imports later on.